src/module/system/statistic/index.ts

```diff
@@ -239,14 +239,14 @@ export class Statistic {
 
     /** Returns data intended to be merged back into actor data */
     getTraceData(this: Statistic, options: RollOptionParameters = {}): StatisticTraceData {
-        const { check } = this.withRollOptions(options);
+        const { check, dc } = this.withRollOptions(options);
 
         return {
             slug: this.slug,
             label: this.label,
-            value: check?.mod ?? 0,
-            totalModifier: check?.mod ?? 0,
-            breakdown: check?.breakdown ?? "",
+            value: dc.value,
```

Comment:
Is this correct? Most uses of value are StatisticModifier are totalModifier rather than the dc. For example, perception.

Reply:
But not AC or (formerly) Class DC. Note how current PCs are showing their "class modifier" because of that change.

Reply:
Got it, its just an inconsistent mess then. At the very skills/perception use the built statistic rather than trace data.

Reply:
Maybe we should have totalModifier and dc in the trace data?

Reply:
I think this change is fine. It includes both now. I just wasn't aware how inconsistent of a mess statistic modifier usage was.